helmfile/charts/networkpolicy-common/values.yaml

```diff
@@ -0,0 +1,40 @@
+fullnameOverride: ""
+nameOverride: ""
+
+global:
+  apiserver:
+    ips:
+      - "0.0.0.0/0"
+    port: 6443
+  ingress:
+    ips:
+      - "0.0.0.0/0"
+  nodes:
+    ips:
+      - "0.0.0.0/0"
```

Comment:
Are these used?

Reply:
No, I think they came with it when I copied over the values from either sc or wc. But I'll keep it and add the if/else in the values template so people can build on it.